Adafruit_Video_Looper/video_looper.py

```diff
@@ -310,7 +355,7 @@ def run(self):
                     # Start playing the first available movie.
                     self._print('Playing movie: {0} {1}'.format(movie, infotext))
                     # todo: maybe clear screen to black so that background (image/color) is not visible for videos with a resolution that is < screen resolution
-                    self._player.play(movie.filename, loop=-1 if playlist.length()==1 else movie.repeats, vol = self._sound_vol)
+                    self._player.play(movie.filename, movie.title, loop=-1 if playlist.length()==1 else movie.repeats, vol = self._sound_vol)
```

Comment:
sounds good to me! the vol option would still only be possible for omx. should we do this in a seperate PR?

Reply:
> another video appears, the program would still be stuck in an endless loop

actually this should not be the case. because if the directory changes either via plugging in a new drive or if the number of video files in the directory changes (this was introduced in one of the recent versions) omxplayer is killed and playlist is rebuilt.

Reply:
OK!